srcs/layouts/latn_bone.xml

```diff
@@ -1,72 +1,131 @@
-<?xml	version="1.0"	encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <!-- https://neo-layout.org/Layouts/bone/ -->
-<keyboard name="Bone" bottom_row="false" script="latin">
-	<!-- first row + characters from number row:
+<keyboard name="Bone" script="latin">
+	<modmap>
+		<!-- shift on number row:
+			1234567890 -> °§ℓ»«$€„“”
+		-->
+		<shift a="1" b="°"/>
+		<shift a="2" b="§"/>
+		<shift a="3" b="ℓ"/>
+		<shift a="4" b="»"/>
+		<shift a="5" b="«"/>
+		<shift a="6" b="$"/>
+		<shift a="7" b="€"/>
+		<shift a="8" b="„"/>
+		<shift a="9" b="“"/>
+		<shift a="0" b="”"/>
+
+		<!-- shift on bottom row:
+		 	,. -> –•
+		-->
+		<shift a="," b="–"/>
+		<shift a="." b="•"/>
+	</modmap>
+	<!-- number row:
+	 	left of numbers:
+			accent_circonflexe/superscript
+			accent_caron/subscript
+			↻
+			accent_dot_above
+			accent_hook_above/accent_horn
+			accent_dot_below
+		numbers:
+			1234567890
+			°§ℓ»«$€„“” -> shift modmap
+			¹²³›‹¢¥‚‘’
+			₁₂₃♀♂⚥ϰ⟨⟩₀v
+
+		right of numbers:
+			accent_grave
+			accent_cedille/accent_ogonek
+			accent_ring
+			accent_trema
+			accent_rough_breathing (not implemented yet)
+			accent_macron/accent_macron_below (not implemented yet)
+	-->
+	<row>
+		<!--left side-->
+		<key width="0.5" key0="loc accent_circonflexe" nw="superscript"/>
+		<key key0="1" ne="¹" se="₁" nw="loc accent_caron" n="subscript"/>
```

Comment:
`₁`, `₂`, `₃` and `₀` are already accessible with `subscript`, which is also present on the layout.
Same for the superscripts.

Reply:
i think they were put there because ¹²³₁₂₂₀ are used more frequently that all the other numbers